so_vector/operations/default.json

```
@@ -521,7 +521,7 @@
   "param-source": "esql-knn-param-source",
   "exact": true,
   "k": 10,
-  "filter": "term(tags, \"java\")"
+  "filter": "match(tags, \"java\")"
```

💭 🔧  
```suggestion
  "filter": "match(tags, \"java\"), {\"boost\": 0})"
```
Do we also need it here?

---

No, on exact search we're not scoring (there's no METADATA _score), so the filter is applied without scoring